.github/workflows/build_agent.yml

```diff
@@ -248,7 +247,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install requests pybase64
```

Comment:
If this is run in our runners, then it should be installed in the runner itself. No need to install every run.